src/libs/TransactionUtils.ts

```diff
@@ -147,6 +147,13 @@ function isMerchantMissing(transaction: Transaction) {
     return isMerchantEmpty && isModifiedMerchantEmpty;
 }
 
+/**
+ * Check if the merchant is partial i.e. `(none)`
+ */
+function isPartialMerchant(merchant: string): boolean {
```

Comment:
Mobile now but is there really no method like this yet?

Reply:
Not yet. What we do though everywhere is to use the equality operator and check for partial merchant text.

Reply:
@mountiny in a follow-up, I could send a PR that replaces all usages with this function.
I'm on the fence if we really need a function for an equality check. but ok

Reply:
> I'm on the fence if we really need a function for an equality check

I had thought about this, but a helper method seemed better from the perspective of code readability. Further, having a central place to handle partial merchant text seems better if any additional text needs to be considered as partial merchant text. 
And I like the idea of replacing the existing usage with this method. We may also want to add a helper method for `isEmptyMerchant` that is duplicated in three places. A follow-up refactor PR seems a good idea.

Reply:
thank you! will create one after this